### PR TITLE
Java User Group Hyderabad event

### PR DESCRIPTION
**Pull request by jbuddha**

### CODE REVIEW

File: _data/events/Hyderabad-JUG.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "title": "JUG Hyderabad GDCR 17",
+  "url": "https://www.meetup.com/jughyderabad/events/244680395",
+  "moderators": [
+    "@jbuddha"
```
Comment (rradczewski) [inline _data/events/Hyderabad-JUG.json]: Are you sure [`@jbuddha`](https://twitter.com/jbuddha) is your twitter handle? :)

Comment (jbuddha) [reply _data/events/Hyderabad-JUG.json]: sorry, I thought we have to give github user id, will submit another pull request

**Comment by jbuddha** [comment]
Fixed twitter handle...

File: _data/events/Hyderabad-JUG.json (hunk 1 above)
Comment (jbuddha) [reply _data/events/Hyderabad-JUG.json]: updated the pull request.please merge. thanks for correction

**Pull request merged**